lib/common/messenger.js

```diff
@@ -263,77 +264,103 @@ function messengerFactory(
         assert.object(data, 'data');
 
         var self = this,
-            subscription,
-            timeout,
-            timer = new Timer();
+	    timer = new Timer();
```

Comment:
These are all indented with tabs instead of spaces, so that should be changed (quick vim: `:%s/\t/    /`)